tests/strategies/external/test_load.py

```diff
@@ -43,3 +47,45 @@ def test_list_bids_succeeds(self, ext_load_fixture):
     def test_delete_bid_succeeds(self, ext_load_fixture):
         check_external_command_endpoint_with_correct_payload_succeeds(ext_load_fixture,
                                                                       "delete_bid", {})
+
+    def test_bid_aggregator(self, ext_load_fixture):
+        ext_load_fixture.state._energy_requirement_Wh[
+            ext_load_fixture.next_market.time_slot] = 1000.0
+        return_value = ext_load_fixture.trigger_aggregator_commands(
+            {
+                "type": "bid",
+                "price": 200.0,
+                "energy": 0.5,
+                "transaction_id": str(uuid.uuid4())
+            }
+        )
+        assert_bid_offer_aggregator_commands_return_value(return_value, False)
+
+    def test_delete_bid_aggregator(self, ext_load_fixture):
+        bid = ext_load_fixture.post_bid(ext_load_fixture.next_market, 200.0, 1.0)
+
+        return_value = ext_load_fixture.trigger_aggregator_commands(
+            {
+                "type": "delete_bid",
```

Comment:
I think `time_slot` being part of it too . Right?

Reply:
These are generic tests for the aggregator commands, I am not testing the time_slot functionality here. This is tested by test_get_market_from_cmd_arg_returns_settlement_market , test_get_market_from_cmd_arg_returns_spot_market tests